### PR TITLE
[python3/ja-jp] New Translation: python3/ja-jp

### PR DESCRIPTION
**Pull request by kakakaya**
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: ja-jp/python3.html.markdown [hunk 1]
```diff
@@ -0,0 +1,897 @@
...
+language: python3
+contributors:
+    - ["Louie Dinh", "http://pythonpracticeprojects.com"]
+    - ["Steven Basart", "http://github.com/xksteven"]
+    - ["Andre Polykanine", "https://github.com/Oire"]
+    - ["Zachary Ferguson", "http://github.com/zfergus2"]
+    - ["evuez", "http://github.com/evuez"]
+translators:
+    - ["kakakaya", "https://github.com/kakakaya"]
+filename: learnpython3.py
```
Comment (vendethiel) [inline ja-jp/python3.html.markdown]: please add `-jp` suffix to filename

File: ja-jp/python3-jp.html.markdown [hunk 2]
```diff
@@ -0,0 +1,897 @@
...
+language: python3
+contributors:
+    - ["Louie Dinh", "http://pythonpracticeprojects.com"]
+    - ["Steven Basart", "http://github.com/xksteven"]
+    - ["Andre Polykanine", "https://github.com/Oire"]
+    - ["Zachary Ferguson", "http://github.com/zfergus2"]
+    - ["evuez", "http://github.com/evuez"]
+translators:
+    - ["kakakaya", "https://github.com/kakakaya"]
+filename: learnpython3.py
```
Comment (vendethiel) [inline ja-jp/python3-jp.html.markdown]: please add the suffix here as well :) ty

**Comment by kakakaya** [comment]
sorry, I've misunderstood.
Is this ok?

**Comment by vendethiel** [comment]
Perfect, thanks!

**Pull request merged**